ansible/roles/dashd/templates/dash.conf.j2

```diff
@@ -78,6 +78,11 @@ sporkaddr={{ dashd_sporkaddr }}
 sporkkey={{ dashd_sporkkey }}
 {% endif %}
 
+{% if dashd_walletnode is not none and coinjoin_wallet_count is defined %}
```

Comment:
What if `coinjoin_wallet_count` is `0`?

Reply:
This condition still doesn't make sense to me, is `coinjoin_wallet_count` per mixing node?

Reply:
I think the condition here should be `dashd_ismixingnode`

Reply:
@ktechmidas this still makes no sense to me

Reply:
As mentioned elsewhere, this whole code is now useless and needs removing...

1) We set the variables on startup of mixing, not in the config file

2) It used to be the wallet node, we had 3 mixing wallets on a wallet node (your initial implementation) which didn't work, so went to using mixing nodes. 

I will remove it